pkg/render/packet_capture_api.go

```diff
@@ -69,6 +77,10 @@ func PacketCaptureAPI(cfg *PacketCaptureApiConfiguration) Component {
 	}
 }
 
+func PacketCaptureAPIPolicy(cfg *PacketCaptureApiConfiguration) Component {
```

Comment:
As discussed, this pattern is intentional since some controllers have multiple components, therefore placing policy at the end of a components Objects() list is not a guarantee that policy is rendered last in the controller.

Will keep as-is, and apply the same pattern when clusterconnection and monitor controllers are adjusted in the immediate follow-up PR.